editor/editor_log.cpp

```diff
@@ -316,6 +346,66 @@ void EditorLog::_reset_message_counts() {
 	}
 }
 
+void EditorLog::_resource_path_clicked(Variant path) {
+	if (path.get_type() == Variant::Type::STRING) {
+		String path_str = static_cast<String>(path);
+
+		PackedStringArray split_by_colon = path_str.split(":");
+		String path_base = split_by_colon[1]; // Used to get the path without line numbers, if the path so happens to have them.
+		String file_type = EditorFileSystem::get_singleton()->get_file_type("res:" + path_base);
+
+		if (file_type == "PackedScene") {
+			// If file is a scene, open.
+			EditorNode::get_singleton()->open_request("res:" + path_base);
+		} else {
+			if (file_type.contains("Script")) {
+				/*
+					Check if file extension contains line number
+					Example:
+					res://path.gd:5
+					(get_extension will return ".gd:5")
+				*/
+
+				if (path_str.get_extension().contains(":")) {
+					EditorNode::get_singleton()->load_resource("res:" + path_base, true);
+
+					/*
+					  split_by_colon
+					  value - index
+
+					  res://path.gd:number:number
+
+					  res - 0
+					  //path.gd - 1
+					  number - 2 (Line Number)
+					  number - 3 (Line Column)
+					*/
+
+					int size = split_by_colon.size() - 1; // Subtracting 1 for readability in if statements below
+
+					if (size >= 2) { // Check if index 2 (Line Number) exists
+						// Go to line
+						ScriptEditor::get_singleton()->call("goto_line", split_by_colon[2].to_int() - 1);
+					}
+
+					if (size >= 3) { // Check if index 3 (Line Column) exists
+						// Go to column
+						ScriptTextEditor *script_editor = Object::cast_to<ScriptTextEditor>(ScriptEditor::get_singleton()->call("get_current_editor"));
+
+						// For some reason, the caret is offset +4 from the value inputed into set_caret_column
+						// Subtract 4 to set the correct column
+						script_editor->get_base_editor()->call_deferred(SNAME("set_caret_column"), split_by_colon[3].to_int() - 4);
```

Comment:
Strange, clicking on `res://new_script.gd:2:8` brings me to `2:5`, but `res://new_script.gd:5:10` works correctly.